Update support_08_preserve_publish.md (#192)
DHP-stottepakke/pages: pages/dmp_supporting_info/support_08_preserve_publish.md

" %}

## About this chapter

> Research data is valuable. Data may be very costly to collect or generate. Making data available for reuse in an ongoing process to build new knowledge, is thus the right thing to do, economically as well as research ethically, to ensure the value of the data is maximised. It is therefore important to follow good practice in how to preserve data and make data accessible for future use. This is done by use of archives (also called repositories) where data is organised in files with formats that we trust to be accessible also long into the future. And moreover, the data need to be described well, so that there is no ambiguity to what information the data holds, including all relevant circumstances and conditions that may have had impact when the data was collected or generated. Description of the data is often done in a so called ‘readme’ file entered along with the data files, in addition to attaching rich metadata that holds information on the data.

Reviewer: ```suggestion
Research data is valuable. Collecting or generating data may require significant effort. Making data available for reuse in an ongoing process to build new knowledge, is thus the right thing to do, economically as well as research ethically, to ensure the value of the data is maximised. It is therefore important to follow good practice in how to preserve data and make data accessible for future use. This is done by use of archives (also called repositories) where data is organised in files with formats that we trust to be accessible also long into the future. And moreover, the data need to be described well, so that there is no ambiguity to what information the data holds, including all relevant circumstances and conditions that may have had impact when the data was collected or generated. Description of the data is often done in a so called ‘readme’ file entered along with the data files, in addition to attaching rich metadata that holds information on the data.
```

> Research data is valuable. Data may be very costly to collect or generate. Making data available for reuse in an ongoing process to build new knowledge, is thus the right thing to do, economically as well as research ethically, to ensure the value of the data is maximised. It is therefore important to follow good practice in how to preserve data and make data accessible for future use. This is done by use of archives (also called repositories) where data is organised in files with formats that we trust to be accessible also long into the future. And moreover, the data need to be described well, so that there is no ambiguity to what information the data holds, including all relevant circumstances and conditions that may have had impact when the data was collected or generated. Description of the data is commonly done in a so called ‘readme’ file entered along with the data files, in addition to attaching rich metadata that holds information on the data.

Reviewer 2: ```suggestion
Research data is valuable. Data may be very costly to collect or generate. Making data available for reuse in an ongoing process to build new knowledge, is thus the right thing to do, economically as well as research ethically, to ensure the value of the data is maximised. It is therefore important to follow good practice in how to preserve data and make data accessible for future use. This is done by use of archives (also called repositories) where data is organised in files with formats that we trust to be accessible also long into the future. And moreover, the data need to be described well, so that there is no ambiguity to what information the data holds, including all relevant circumstances and conditions that may have had impact when the data was collected or generated. Description of the data is often done in a so called ‘readme’ file entered along with the data files, in addition to attaching rich metadata that holds information on the data.
```

> Research data is valuable. Data may be very costly to collect or generate. Making data available for reuse in an ongoing process to build new knowledge, is thus the right thing to do, economically as well as research ethically, to ensure the value of the data is maximised. It is therefore important to follow good practice in how to preserve data and make data accessible for future use. This is done by use of archives (also called repositories) where data is organised in files with formats that we trust to be accessible also long into the future. And moreover, the data need to be described well, so that there is no ambiguity to what information the data holds, including all relevant circumstances and conditions that may have had impact when the data was collected or generated. Description of the data is commonly done in a so called ‘readme’ file entered along with the data files, in addition to attaching rich metadata that holds information on the data.

Reviewer 2: ```suggestion
Research data is valuable. Data may be very costly to collect or generate. Making data available for reuse in an ongoing process to build new knowledge, is thus the right thing to do, economically as well as research ethically, to ensure the value of the data is maximised. It is therefore important to follow good practice in how to preserve data and make data accessible for future use. This is done by use of archives (also called repositories) where data is organised in files with formats that we trust to be accessible also long into the future. And moreover, the data need to be described well, so that there is no ambiguity to what information the data holds, including all relevant circumstances and conditions that may have had impact when the data was collected or generated. Description of the data is often done in a so called ‘readme’ file entered along with the data files, in addition to attaching rich metadata that holds information on the data.
```

This chapter will assist you in walking through the planning steps of making the data available in line with the FAIR principles, including selecting an archive to publish the data (or, if needed, archiving with access restrictions) and providing appropriate metadata. The chapter includes both questions that should be considered at the initial stage of the DMP, and questions to be answered towards the end of the research project to plan the fate of data in the project, whether they will be deposited in a research data archive, preserved in other ways, or in certain cases will be discarded.

### Trustworthy research data archives

> The process of selecting archive to use is therefore important, to ensure best archival practice and the quality of the data. The most important issue to look for is whether the archive is a trustworthy one, that is either certified with e.g. the CoreTrustSeal or the ISO 16363 standard. A prerequisite for a trusted archive is the curation procedure. Curation should be done in cooperation between archive curators and the depositing researcher to ensure the data is preserved in preferred file formats and accompanied by good description and rich metadata.

Reviewer 2: I suggest to lean on the HE AGA criteria p.373f https://ec.europa.eu/info/funding-tenders/opportunities/docs/2021-2027/common/guidance/aga_en.pdf

```suggestion
The process of selecting archive to use is therefore important, to ensure best archival practice and the quality of the data. The most important issue to look for is whether the archive is a trustworthy one, that is either certified with e.g. the CoreTrustSeal or the ISO 16363 standard or are disciplinary or domain repositories that are widely endorsed and supported by research communities and are recognized on an international scale.  Prerequisites for trusted archives include a curation procedure. Curation should be done in cooperation between archive curators and the depositing researcher to ensure the data is preserved in preferred file formats and accompanied by good description and rich metadata.
```

You may browse or search for suitable archives in the [re3data registry](https://www.re3data.org/), which is a registry of data archives. The [FAIRsharing service](https://fairsharing.org/) is also a useful a searchable resource that holds information on data archives, as well as standards and data policies. Both resources have filtering functions.

You are advised to choose a so-called [trustworthy archive](/pages/support_00_rdm_terms#trustworthy-repository-trustworthy-archive) for the data that meets minimum criteria on provision of persistent and unique identifiers (PIDs), metadata, data access and usage licenses, and data preservation. A common certification is the [Core Trust Seal](https://www.coretrustseal.org/) which certifies that the archive in question operates in accordance with the Core Trust Seal quality criteria, both in terms of its financial foundation and its quality assuring data curation routines.

> Data should be archived as open as possible and as closed as necessary, as is the guiding principle commonly referred to by research funders and others. This means that data archiving does not necessarily mean to make data openly available. When archiving data in a certified archive, the data is preserved for the future according to best practice. The issue of publishing data is a separate question. Remember: Make your data as open as possible, and as closed as necessary. See Chapter 3 (https://plan.research-data.no/pages/support_03_legal_ethics) of these guidelines for more on the issue of data that needs protection from public access. And this too is an important issue when selecting where to archive the data: If data need access control, which archive is well suited for this?

Reviewer: ```suggestion
Data should be archived **as open as possible and as closed as necessary**, as is the guiding principle commonly referred to by research funders and others. This means that data archiving does not necessarily mean to make data openly available. When archiving data in a certified archive, the data is preserved for the future according to best practice. The issue of publishing data is a separate question. See Chapter 3 (https://plan.research-data.no/pages/support_03_legal_ethics) of these guidelines for more on the issue of data that needs protection from public access. And this too is an important issue when selecting where to archive the data: If data need access control, which archive is well suited for this?
```

> Data should be archived as open as possible and as closed as necessary, as is the guiding principle commonly referred to by research funders and others. This means that data archiving does not necessarily mean to make data openly available. When archiving data in a certified archive, the data is preserved for the future according to best practice. The issue of publishing data is a separate question. Remember: Make your data as open as possible, and as closed as necessary. See Chapter 3 (https://plan.research-data.no/pages/support_03_legal_ethics) of these guidelines for more on the issue of data that needs protection from public access. And this too is an important issue when selecting where to archive the data: If data need access control, which archive is well suited for this?

Reviewer: ```suggestion
Data should be archived as open as possible and as closed as necessary, as is the guiding principle commonly referred to by research funders and others. This means that data archiving does not necessarily mean to make data openly available. When archiving data in a certified archive, the data is preserved for the future according to best practice. The issue of publishing data is a separate question. Remember: Make your data as open as possible, and as closed as necessary. See [Chapter 3](/pages/support_03_legal_ethics) of these guidelines for more on the issue of data that needs protection from public access. And this too is an important issue when selecting where to archive the data: If data need access control, which archive is well suited for this?
```

### Restricted access to research data

> Some data should not be made openly available, according to law regulations (including the General Data Protection Regulation (GDPR) and/or ethical considerations. These may be data with person identifying information, or data that are sensitive for other reasons (national security or business information). It may still be important to ensure their long term preservation and regulate data access beyond the project period. In many cases data can be archived with restricted access, enabling access only to eligible actors.

Reviewer: ```suggestion
Some data should not be made openly available, according to law regulations (including the General Data Protection Regulation (GDPR) and/or ethical considerations. These may be data with person identifying information, or data that are [sensitive for other reasons](/pages/support_03_legal_ethics#sensitive-data) including national security or intellectual property rights. It may still be important to ensure their long term preservation and regulate data access beyond the project period. In many cases data can be archived with restricted access, enabling access only to eligible actors.
```

" %}

## About this chapter

> Research data is valuable. Collecting or generating data may require significant effort. Making data available for reuse in an ongoing process to build new knowledge, is thus the right thing to do, economically as well as research ethically, to ensure the value of the data is maximised. It is therefore important to follow good practice in how to preserve data and make data accessible for future use. This is done by use of archives (also called repositories) where data is organised in files with formats that we trust to be accessible also long into the future. And moreover, the data need to be described well, so that there is no ambiguity to what information the data holds, including all relevant circumstances and conditions that may have had impact when the data was collected or generated. Description of the data is often done in a so called ‘readme’ file entered along with the data files, in addition to attaching rich metadata that holds information on the data.

Reviewer 2: ```suggestion
Research data is valuable. Collecting or generating data may require significant effort. Making data available for reuse in an ongoing process to build new knowledge, is thus the right thing to do, economically as well as research ethically, to ensure the value of the data is maximised. It is therefore important to follow good practice in how to preserve data and make data accessible for future use. This is done by use of archives (also called repositories) where data is organised in files with formats that we trust to be accessible also long into the future. And moreover, the data need to be described well, so that there is no ambiguity to what information the data holds, including all relevant circumstances and conditions that may have had impact when the data was collected or generated. Description of the data is often done in a so called ‘README’ file entered along with the data files, in addition to attaching rich metadata that holds information on the data.
```
